docs/start/getting-started/fragments/vanillajs/data-model.md

```diff
@@ -163,32 +170,51 @@ After restarting your app using `npm start` go back to your browser and click **
 +const QueryResult = document.getElementById("QueryResult");
 
  MutationButton.addEventListener("click", (evt) => {
-@@ -19,3 +30,5 @@ MutationButton.addEventListener("click", (evt) => {
+   createNewTodo().then((evt) => {
      MutationResult.innerHTML += `<p>${evt.data.createTodo.name} - ${evt.data.createTodo.description}</p>`;
    });
  });
-+
+
 +getData();
```

Comment:
We should also removes these erroneous `+` signs throughout the code.

Reply:
plus signs get rendered in the docs as "green lines". this is intended.